packages/react-components/react-infobutton/README.md

```diff
@@ -1,29 +1,25 @@
 # @fluentui/react-infobutton
 
-**React Infobutton components for [Fluent UI React](https://react.fluentui.dev/)**
-
-## STATUS: WIP 🚧
-
-These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
+**React InfoLabel components for [Fluent UI React](https://react.fluentui.dev/)**
```

Comment:
hmm maybe we should create a new package that will include InfoLabel only ? 

with this approach this is a bit confusing. react-infobutton package exposes only `InfoLabel` , why is it called infobutton then ?

wdyt ?

Reply:
I think that's fair, the only thing is that this package was created before the unstable exports issues. So InfoLabel and InfoButton will still be under unstable and we would have to deprecate both. I can definitely make the `react-infolabel` package, but with those concerns, what are your thoughts?

Reply:
After seeing the reply to my comment above, I see why this makes sense. I do want to know what your thoughts on basically having an unusable react-infobutton in our repo and react-components? We would create this second package, but react-infobutton would stay there with no purpose, except to not cause a breaking change.